.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       # https://github.com/KDAB/cxx-qt/issues/121
       #
       # automake is needed for building libicu which is a dependency of Qt
-      run: brew install automake autoconf-archive ninja clang-format
```

Comment:
we should instead be removing clang-format from the ctests as well and running clang-format in it's own job, see clang-format-rs for ideas https://github.com/KDAB/clang-format-rs/blob/main/.github/workflows/github-clang-format-tests.yml